src/views/product.js

```diff
@@ -13,6 +13,10 @@ export default class ProductView extends View {
     return true;
   }
 
+  get outerHeight() {
+    return `${this.wrapper.clientHeight}px`;
+  }
+
```

Comment:
is there a usage of this prop anywhere in the project? And should there be a test?

Reply:
It's used in `resizeY()` within `view.js` during the `render` process. https://github.com/Shopify/buy-button-js/blob/ba89d7fa2f4944da7f388a9b6d632d764b376e5f/src/view.js#L270

In terms of tests, it seems custom `view` classes aren't really tested (I was following how the `toggle` component's custom `outerHeight()` was implemented and that didn't have test coverage either).
https://github.com/Shopify/buy-button-js/blob/ba89d7fa2f4944da7f388a9b6d632d764b376e5f/src/views/toggle.js#L22

Reply:
huh. I'm curious how the usage in `view.js` works if this hasn't been merged yet.

Also, I'm going to question two things:

If this is just an alias for `this.wrapper.clientHeight` why doesn't the caller just access `this.wrapper.clientHeight`. Adding it as an alias makes it a public method that needs to be documented and supported.

Second: Without a clear expectation in a test for this alias to exist _or_ for the behaviour that it enables, it's quite hard to understand why we would need it.

Reply:
Just added a test for product view `outerHeight` property 😄 

>I'm also curious now though, is the computed style ever worthwhile? What do you miss out on by just defaulting to this.wrapper.clientHeight everywhere?

Good question - TBH, I'm not entirely sure how defaulting to `this.wrapper.clientHeight` everywhere would affect the individual components. @harisaurus - do you have any insight on this?  I think for now, we can use the override solution to fix the immediate FF issue and address this in a future PR  since updating the `view.outerHeight` will need extensive testing.